README.md

```diff
@@ -59,43 +83,40 @@ The canonical download instructions for IPFS are over at: https://docs.ipfs.io/g
 
 ### System Requirements
 
-IPFS can run on most Linux, macOS, and Windows systems. We recommend running it on a machine with at least 2 GB of RAM and 2 CPU cores (go-ipfs is highly parallel). On systems with less memory, it may not be completely stable.
+IPFS can run on most Linux, MacOS, and Windows systems. We recommend running it on a machine with at least 2 GB of RAM and 2 CPU cores (go-ipfs is highly parallel). On systems with less memory, it may not be completely stable.
 
 If your system is resource-constrained, we recommend:
 
 1. Installing OpenSSL and rebuilding go-ipfs manually with `make build GOTAGS=openssl`. See the [download and compile](#download-and-compile-ipfs) section for more information on compiling go-ipfs.
 2. Initializing your daemon with `ipfs init --profile=lowpower`
 
-### Install prebuilt packages
-
-We host prebuilt binaries over at our [distributions page](https://ipfs.io/ipns/dist.ipfs.io#go-ipfs).
+### Docker
 
-From there:
-- Click the blue "Download go-ipfs" on the right side of the page.
-- Open/extract the archive.
-- Move `ipfs` to your path (`install.sh` can do it for you).
+[![Docker Image Version (latest semver)](https://img.shields.io/docker/v/ipfs/go-ipfs?color=blue&label=go-ipfs%20docker%20image&logo=docker&sort=semver&style=for-the-badge&cacheSeconds=3600)](https://hub.docker.com/r/ipfs/go-ipfs/)
 
-You can also download go-ipfs from this project's GitHub releases page if you are unable to access ipfs.io.
+More info on how to run go-ipfs inside docker can be found [here](https://docs.ipfs.io/how-to/run-ipfs-inside-docker/).
 
-### From Linux package managers
+### Native Linux package managers
 
 - [Arch Linux](#arch-linux)
-- [Nix](#nix)
-- [Solus](#solus)
-- [Snap](#snap)
+- [openSUSE](#openSUSE)
 
-#### Arch Linux
+#### ArchLinux
 
-In Arch Linux go-ipfs is available as
-[go-ipfs](https://www.archlinux.org/packages/community/x86_64/go-ipfs/) package.
+[![go-ipfs via Community Repo](https://img.shields.io/archlinux/v/community/x86_64/go-ipfs?color=1793d1&label=go-ipfs&logo=arch-linux&style=for-the-badge&cacheSeconds=3600)](https://wiki.archlinux.org/title/IPFS)
 
-```
-$ sudo pacman -S go-ipfs
-```
+[![go-ipfs-git via AUR](https://img.shields.io/aur/version/go-ipfs-git?color=1793d1&label=go-ipfs-git&logo=arch-linux&style=for-the-badge&cacheSeconds=3600)](https://aur.archlinux.org/packages/go-ipfs-git/)
```

Comment:
I wouldn't use the AUR badge here. Git PKGBUILDs are rarely updated, so the version will displayed here fall behind.

Reply:
Well, I'm the maintainer. I can update it from time to time :)